added multi-line test

Neon22 opened the pull request:
with color.

Neon22 committed:
added multi-line test

with color

Neon22 commented on rtflib/tests/helloworld-multi.rtf:
```diff
@@ -0,0 +1,17 @@
+\red0\green255\blue0;
+}
+hello world. A Line of text.\
+\ 
```
not sure if there's a better way to format these so output has less line breaks...

viking-sudo-rm replied:
You can open an issue documenting this if you're not sure how to fix it but think it might be worth addressing later (or worth someone else addressing)

Neon22 replied:
will do

viking-sudo-rm commented on rtflib/tests/test_rtflib.py:
```diff
@@ -19,6 +19,26 @@
 }
 """
 
+HELLO_WORLD_MULTI = """
```
Can you remove all these variables in the test case (`HELLO_WORLD`, `HELLO_WORLD_MULTI`, `TABLE`, etc.)?

I replaced them with rtf files so they are unused now.

Neon22 committed:
Update test_rtflib.py

Neon22 committed:
format bug

use of `self.format.size` instead of `self.size`

Also added strikethrough option

Neon22 committed:
updated multi test to include format

Neon22 commented:
OK. stopping in this branch:
- got format working 
- updated muti-line example to include a format change
- added strikethrough format option
